src/components/Header/Header.jsx

```diff
@@ -103,30 +105,39 @@ class Header extends Component {
       } else {
         window.location = returnUrl;
       }
+    } else {
+      window.location = returnUrl || 'https://3dlook.ai/mobile-tailor/';
     }
```

Comment:
make it const e.g.

const DEFAULT_RETURN_URL = 'https://3dlook.ai/mobile-tailor/';